packages/interface/src/features/applications/components/ReviewApplicationDetails.tsx

```diff
@@ -122,6 +126,34 @@ export const ReviewApplicationDetails = (): JSX.Element => {
           title="Funding sources"
         />
       </div>
+
```

Comment:
Can you replace this part to using the `ProjectItem`? and since it's not submitted yet (as a `IRecipient`), maybe we could slightly change the input to make it fit? since the duplicated code is hard to maintain

Reply:
Thanks! added the fixes